test/serializer/additional-functions/modifiedobjects.js

```diff
@@ -32,6 +32,5 @@ inspect = function inspect() {
   let z2 = toCapture1;
   additional2();
 
-  return 'PASS';
-  //return '' + z + z2 + x + y + toCapture1 + toCapture2 + toCapture3 + foo + bar + (foo === bar);
+  return '' + z + z2 + x + y + toCapture1 + toCapture2 + toCapture3 + foo + bar + (foo === bar);
```

Comment:
I think we are testing wrong direction here. We should test the nested function inside additional function that modifies free variables instead of additional function itself. For example, can you try if the following testcase pass?

```js
(function () {
  let top = 5;
  function af1() {
    let mutable = 3;
    return function() {
      return (++mutable) + (--top);
    }
  };
  global.residual = function() {
    return ++top;
  };
  global.f = af1;
  global.__registerAdditionalFunctionToPrepack(af1);
  inspect = function() {
    return f()() + residual();
  }
})();
```